server-external.js

```diff
@@ -71,6 +71,12 @@ async function start () {
       options: {
         reauthenticate: connectors.idm.users.reauthenticate.bind(connectors.idm.users)
       }
+    }, {
+      plugin: require('shared/plugins/error'),
+      options: {
+        logger,
+        contextDefaults: require('./src/external/lib/view').contextDefaults
```

Comment:
Is it still needed to pass these in? 
It might be possible to just use request.view in the error plugin handler now.